src/schematools/importer/ndjson.py

```diff
@@ -86,13 +86,13 @@ def parse_object(self, source: dict) -> dict[str, list[Record]]:
         sub_rows = {}
 
         self._fix_through_fields(source)
-        self._fill_composite_pk(source, main_row)
+        composite_key_set = self._fill_composite_pk(source, main_row)
```

Comment:
De `_set` in de naam suggereert dat het een python `set` is. Wellicht iets als `composite_key_filled` oid van maken?

Reply:
Veranderd.